dev/README_RELEASE_PROVIDERS.md

```diff
@@ -1283,6 +1136,38 @@ If you want to disable this behaviour, set the env **CLEAN_LOCAL_TAGS** to false
 breeze release-management tag-providers
 ```
 
+## Publish documentation
+
+Documentation is an essential part of the product and should be made available to users.
+In our cases, documentation for the released versions is published in S3 bucket, and the site is
+kept in a separate repository - [`apache/airflow-site`](https://github.com/apache/airflow-site),
+but the documentation source code and build tools are available in the `apache/airflow` repository, so
+you need to run several workflows to publish the documentation. More details about it can be found in
+[Docs README](../docs/README.md) showing the architecture and workflows including manual workflows for
+emergency cases.
+
+There are two steps to publish the documentation:
+
+1. Publish the documentation to S3 bucket.
+
+The release manager publishes the documentation using GitHub Actions workflow
+[Publish Docs to S3](https://github.com/apache/airflow/actions/workflows/publish-docs-to-s3.yml).
+
+You can specify the tag to use to build the docs and list of providers to publish (separated by spaces) or
+``all-providers`` in case you want to publish all providers (optionally you can exclude
+some of those providers)
+
+After that step, the provider documentation should be available under the usual urls (same as in PyPI packages)
+but stable links and drop-down boxes should not be updated.
```

Comment:
We should describe on which S3 path we do that.
As far as i remember you asked to do it on stage, review the docs and only if looks good rerun on live?

Reply:
We do not have yet "staging" workflow enabled. Live is the only one that works - we will update it later, but live should be used only for now. I will add it.